src/pages/admin/materiaalaanvragen/index.tsx

```diff
@@ -182,31 +162,37 @@ const AdminMaterialRequests: NextPage<DefaultSeoInfo> = ({ url }) => {
 		);
 	};
 
-	const onTypeClick = (id: string | number): void => {
-		const showAll = id === ADMIN_MATERIAL_REQUESTS_FILTER_ALL_ID;
-		setTypeDropdownLabel(ADMIN_MATERIAL_REQUEST_TYPE_FILTER_RECORD[id]);
-		setIsTypeDropdownOpen(false);
+	const onSearch = (value: string | undefined) => {
+		setFilters({ search: value, page: 1 });
+	};
 
-		setFilters({ ...filters, type: showAll ? '' : `${id}`, page: 1 });
+	const onMultiTypeChange = (selectedItems: string[]) => {
+		if (selectedItems !== selectedTypes) {
```

Comment:
Ik zou hier effectief checken of de values gelijk zijn of niet. 

```
const arr1 = ["id-1", "id-2", "id-3"];
const arr2 = ["id-1", "id-3", "id-2"];
const arr3 = ["id-1", "id-6", "id-3", "id-2"];

// arrays vergelijken
console.log(arr1 === arr2); // false
console.log(arr1 === arr3); // false

// values in arrays vergelijken (https://lodash.com/docs/4.17.15#xorBy)
console.log(xorBy(arr1, arr2)); // []
console.log(xorBy(arr1, arr3)); // ["id-6]
```

Je zou dus zoiets kunnen doen:
```
const diff = xorBy(selectedItems, selectedTypes);

if (diff.length) {
   setSelectedTypes(selectedItems)
}
```